src/main/scala/org/broadinstitute/hail/driver/ExportPlink.scala

```diff
@@ -3,17 +3,50 @@ package org.broadinstitute.hail.driver
 import org.apache.spark.RangePartitioner
 import org.apache.spark.storage.StorageLevel
 import org.broadinstitute.hail.Utils._
+import org.broadinstitute.hail.annotations.Querier
+import org.broadinstitute.hail.io.annotators.SampleFamAnnotator
 import org.broadinstitute.hail.methods.ExportBedBimFam
-import org.broadinstitute.hail.variant.Variant
+import org.broadinstitute.hail.variant.{Variant, VariantDataset}
 import org.kohsuke.args4j.{Option => Args4jOption}
+import org.broadinstitute.hail.expr.{Parser,TBoolean}
 
 object ExportPlink extends Command {
 
   class Options extends BaseOptions {
     @Args4jOption(required = true, name = "-o", aliases = Array("--output"),
       usage = "Output file base (will generate .bed, .bim, .fam)")
     var output: String = _
+
+    @Args4jOption(required = false, name = "-r", aliases = Array("--fam-root"),
+      usage = "Root of fam annotation in the samples. Annotations are expected to be in the same structure as created by annotatesamples fam. If not, field can be overriden with -r* options.")
+    var fr: String = ""
+
+    @Args4jOption(required = false, name = "-rFamID", aliases = Array("--famID-root"),
+      usage = "Root of Family ID annotation in the sample annotation. No need to provide if -r is used.")
+    var rFamID: String = ""
+
+    @Args4jOption(required = false, name = "-rPID", aliases = Array("--paternalID-root"),
+      usage = "Root of Paternal ID annotation in the sample annotation. No need to provide if -r is used.")
```

Comment:
I think "root" is the wrong word for this -- it's really a query / computation.  Root is used in annotation to indicate that a struct is being added to a certain point in the annotation tree